content/tracing/setup/dotnet.md

```diff
@@ -52,25 +52,107 @@ net start w3svc
 
 .NET Core doesn't support the Windows Global Assembly Cache (GAC) used by the .NET Framework. Instead, you can deploy Datadog's managed assemblies together with your application by adding the `Datadog.Trace.ClrProfiler.Managed` NuGet package. It is important that you match the versions of the NuGet package and the MSI installer.
 
-For example, if you are using the MSI installer for version `0.4.1-beta`, use:
+For example, if you are using the MSI installer for version `0.5.0-beta`, use:
 
 ```
-dotnet add package Datadog.Trace.ClrProfiler.Managed --version 0.4.1-beta
+dotnet add package Datadog.Trace.ClrProfiler.Managed --version 0.5.0-beta
 ```
 
 ### Linux
 
-Automatic instrumention for .NET Core on Linux is coming soon. To instrument your .NET Core applications on Linux today, use [manual instrumentation][8].
+Automatic instrumentation is available on Linux for .NET Core 2.0+. To set up automatic instrumentation, 4 steps are necessary:
+
+#### Install the Datadog Agent
+
+Traces are sent to a local Datadog Agent and then forwarded to Datadog. To install the agent, follow the [Linux instructions][1].
+
+#### Add the `Datadog.Trace.ClrProfiler.Managed` NuGet Package
+
+To add the `Datadog.Trace.ClrProfiler.Managed` NuGet package to your project run the following:
+
+```bash
+dotnet add package Datadog.Trace.ClrProfiler.Managed --version 0.5.0-beta
+```
+
+#### Install the `Datadog.Trace.ClrProfiler.Native` Library
+
+Automatic instrumention in Linux is implemented using a shared library available from the `dd-trace-csharp` [releases page][10].
+
+For Debian or Ubuntu, download and install the Debian package:
+
+```bash
+curl -LO https://github.com/DataDog/dd-trace-csharp/releases/download/v0.5.0-beta/datadog-dotnet-apm_0.5.0_amd64.deb
+sudo apt install ./datadog-dotnet-apm_0.5.0_amd64.deb
+```
+
+For CentOS or Fedora, download and install the RPM package
+
+```bash
+curl -LO https://github.com/DataDog/dd-trace-csharp/releases/download/v0.5.0-beta/datadog-dotnet-apm-0.5.0-1.x86_64.rpm
+sudo rpm -Uvh datadog-dotnet-apm-0.5.0-1.x86_64.rpm
+```
+
+A Tar archive is available for other distributions:
+
+```bash
+sudo mkdir -p /opt/datadog
+curl -L https://github.com/DataDog/dd-trace-csharp/releases/download/v0.5.0-beta/datadog-dotnet-apm-0.5.0.tar.gz \
+| sudo tar xzf - -C /opt/datadog
+```
+
+For Alpine Linux you will also need to install `libc6-compat`
+
+```bash
+apk add libc6-compat
+```
+
```

Comment:
This might be a good place to use tabs. [See here for reference](https://github.com/DataDog/documentation/blob/f885a792aed60860ec6522b0486e6160e55bdc56/content/agent/faq/agent-commands.md#L20)